### PR TITLE
315 careerchangers

### LINKED ISSUE #315: Career changers landing page
As a marketer 
So that I can target content to a segmented audience
I want a specific landing page just for career changers

### PR DESCRIPTION
**Pull request by Arfah**
LANDING PAGE!

Closes #315

### CODE REVIEW

File: source/career-changers.html.haml [hunk 1]
```diff
@@ -0,0 +1,96 @@
+- description "Makers Academy is London's leading Web Developer Bootcamp. Learn to code in 12 Weeks by enrolling on one of our programming / web development courses! Learn Ruby, JavaScript and the best software practices."
+%section.hero-splash{style: "background-image: url(#{image_path("backgrounds/dan-and-students-overhead-70.jpg")});"}
+  .container
+    .centered-row
+      %h1 Switch Careers by learning to code in 12 weeks
```
Comment (pitchinvasion) [inline source/career-changers.html.haml]: Careers -> careers

File: source/career-changers.html.haml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+      %p The technology sector has been growing rapidly for years, and has been totally unaffected by the recession. With demand for developers growing 20% year-on-year, and supply of developers (mainly graduates of computer science degrees) flatlining at 1%, the tech industry is crying out for talent. In short - the tech industry needs you!
+      %p Technology companies are well known for their fun offices and relaxed work environment. Our office at Makers Academy has everything from beanbags & table tennis to a hammock & board games! The culture in tech companies is friendly, egalitarian and truly meritocratic, so say goodbye to your stuffy suits - most people tend to wear casual clothes! You’ll also almost certainly have more freedom and flexibilty than your current role... And you get to build cool stuff as well!
```
Comment (pitchinvasion) [inline source/career-changers.html.haml]: change last sentence-> current role&hellip; and you get to build cool stuff as well!

File: source/career-changers.html.haml [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+      %p When was the last time you were so excited by your work that you jumped out of bed in the morning? Never?! Then quit now and make a change for the better - life’s too short not to be doing something you love.
+
+= partial :"full_graduates", locals: { category: "career changer" }
+
+%section.hero{style: "background-image: url(#{image_path("backgrounds/students-in-lecture.jpg")});"}
+  .container
+    .centered-row
+      %h4.leader Meet
+      %h2 Our hiring partners
+      %p sThe average salary for a Makers Academy graduate as a junior developer is £32k, and after a few years experience that number jumps up to £45,000. Compare that to the UK average of £26,500 and you can see how lucrative a career coding can be.
```
Comment (pitchinvasion) [inline source/career-changers.html.haml]: sThe -> The

**Comment by Arfah** [comment]
@sjmog it would good if you can cast your design eye over it. I want to launch this soon.. eg: tomorrow.
THANKS

File: source/employers.html.haml [hunk 4]
```diff
@@ -34,18 +34,18 @@
...
+          Learning to be a developer at Makers Academy is less about
           %em what you know
-          and more about 
+          and more about
           %em how you do it.
-      %p 
-        %em Attitude - 
+      %p
+        %em Attitude -
         Students learn how to solve problems for themsleves, to find their own answers.
```
Comment (roidriscoll) [inline source/employers.html.haml]: *themselves

**Comment by Arfah** [comment]
CAN WE SHIP IT NOW.... please... pretty please....

**Comment by pitchinvasion** [comment]
OK :shipit: there's your squirrel

**Comment by roidriscoll** [comment]
:shipit:

**Pull request merged**